website/addons/googledrive/client.py

```diff
@@ -113,3 +113,12 @@ def folders(self, folder_id='root'):
             throws=HTTPError(401)
         )
         return res.json()['items']
+
+    def file_or_folder_metadata(self, fileId='root'):
```

Comment:
This method doesn't appear to be inherited from anything; please add a docstring (including parameter names and types).